app/ForkedStar.tsx

```diff
@@ -0,0 +1,76 @@
+import octokit from "@/lib/octokit";
+import { env } from "@/env.mjs";
+import { GoRepoForked } from "react-icons/go";
+import { GoStar } from "react-icons/go";
+import { GoPeople } from "react-icons/go";
+import { getContributors } from "@/lib/api";
+
+const GITHUB_ORG: string = env.NEXT_PUBLIC_GITHUB_ORG;
+
+export default async function ForkedStar() {
```

Comment:
Can we rename the file and component to CommunityEngagementSummary so that it's more abstract and we are also showing no. of contributors.

```suggestion
export default async function CommunityEngagemet() {
```